### PR TITLE
Improves #1644 - Add integration tests for /v2/info.

### PR DESCRIPTION
**Pull request by gkleiman**
This PR is based on #1758. I can rebase it on top of master once #1758 is merged.

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package mesosphere.marathon.integration
+
+import mesosphere.marathon.integration.setup._
+import org.scalatest.{ GivenWhenThen, Matchers }
+import play.api.libs.json.Json
+
+class InfoIntegrationTest extends IntegrationFunSuite with SingleMarathonIntegrationTest with GivenWhenThen with Matchers {
+  test("upload and fetch an artifact") {
```
Comment (aquamatthias) [inline src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala]: copy & paste error

File: src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    Then("the response should be successful")
+    response.code should be (200)
+
+    val info = response.entityJson
+
+    And("the http port should be correct")
+    (info \ "http_config" \ "http_port") should be (Json.toJson(config.marathonBasePort))
+
+    And("the ZooKeeper info should be correct")
+    (info \ "zookeeper_config" \ "zk") should be (Json.toJson(config.zk))
```
Comment (aquamatthias) [inline src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala]: do you need the parentheses?

**Comment by aquamatthias** [comment]
LGTM.

**Comment by aquamatthias** [comment]
I wait for the other PR to get merged.

File: src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala (hunk 1 above)
Comment (gkleiman) [inline src/test/scala/mesosphere/marathon/integration/InfoIntegrationTest.scala]: Good catch =)

**Comment by gkleiman** [comment]
Addressed the comments. This PR doesn't depend on  #1758 anymore.

**Pull request closed without merging**